packages/wallet/src/cip30.ts

```diff
@@ -76,7 +91,7 @@ const MAX_COLLATERAL_AMOUNT = CML.BigNum.from_str('5000000');
 
 const cslToCbor = (csl: CslInterface) => Buffer.from(csl.to_bytes()).toString('hex');
 
-const compareUtxos = (utxo: Cardano.Utxo, comparedTo: Cardano.Utxo) => {
+export const compareUtxos = (utxo: Cardano.Utxo, comparedTo: Cardano.Utxo) => {
```

Comment:
Please omit this when re-exporting from wallet/src/index.ts
Otherwise it needs to be tested (and probably hoisted to a more appropriate place)